### PR TITLE
chore: remove flannel beta designation, make default cni

### PR DESCRIPTION
**Pull request by emosbaugh**
Removes Flannel beta designation.

Refers to Flannel in example specs rather than Weave in preparation for making Flannel the default CNI.

### CODE REVIEW

**Review by laverya — APPROVED** [review]

File: src/markdown-pages/introduction/what-kurl-does.md [hunk 1]
```diff
@@ -30,12 +30,12 @@ kURL will perform the following steps on the host prior to delegating to `kubead
...
+Once kubeadm gets the cluster running, it’s not ready for an application yet. A cluster will need networking, storage and more. These services are provided by other open source components, including a lot of the CNCF ecosystem. In a kURL installation manifest, you can specify the additional add-ons that are installed after kubectl starts the cluster. For example, you can include Flannel for a CNI plugin, Rook for distributed storage, Prometheus for monitoring and Fluentd for log aggregation. In addition to specifying the add-ons and versions, most add-ons include advanced options that allow you to specify the initial configuration.
```
Comment (rrpolanco) [inline src/markdown-pages/introduction/what-kurl-does.md:33]:
```suggestion
Once kubeadm gets the cluster running, it’s not ready for an application yet. A cluster will need networking, storage and more. These services are provided by other open source components, many of them from the CNCF ecosystem. In a kURL installation manifest, you can specify the additional add-ons that are installed after kubectl starts the cluster. For example, you can include Flannel for a CNI plugin, Rook for distributed storage, Prometheus for monitoring and Fluentd for log aggregation. In addition to specifying the add-ons and versions, most add-ons include advanced options that allow you to specify the initial configuration.
```

**Comment by rrpolanco** [comment]
Will this change also show Flannel as the default CNI plugin selection on the webpage?

**Comment by emosbaugh** [comment]
> Will this change also show Flannel as the default CNI plugin selection in the homepage?

This will affect what is returned as the default https://github.com/replicatedhq/kURL-api/pull/132/files#diff-642326ae47571da6c4d7db3e0de55fec30d4893f12e9556f317593e9c9a56081R818

**Review by rrpolanco — DISMISSED** [review]

File: src/markdown-pages/introduction/what-kurl-does.md (hunk 1 above)
Comment (rrpolanco) [reply src/markdown-pages/introduction/what-kurl-does.md:33]: looks like this suggestion got lost with the latest commit

Comment (rrpolanco) [inline src/markdown-pages/introduction/what-kurl-does.md:33]:
```suggestion
Once kubeadm gets the cluster running, it’s not ready for an application yet. A cluster will need networking, storage and more. These services are provided by other open source components, many of them from the CNCF ecosystem. In a kURL installation manifest, you can specify the additional add-ons that are installed after kubectl starts the cluster. For example, you can include Flannel for a CNI plugin, Rook for distributed storage, Prometheus for monitoring and Fluentd for log aggregation. In addition to specifying the add-ons and versions, most add-ons include advanced options that allow you to specify the initial configuration.
```

**Review by rrpolanco — APPROVED** [review]

**Review by camilamacedo86 — APPROVED** [review]

**Pull request merged**